openshift/machine-config-operator
# Watch scheduler CR and make masters schedulable accordingly

**- What I did**
Added  logic in node controller to watch for scheduler CR changes and update the master nodes to make them schedulable and unschedulable

**- How to verify it**

~~I did not test it. I want to check if the approach seems reasonable before I proceed further. Please review the https://github.com/openshift/machine-config-operator/commit/c82786d5061928080031eb83274d94601451cf4d~~

The e2e test verifies if the worker label has been added and master taint has been removed for all the masters when scheduler CR's `mastersSchedulable` field has been set or unset. That field was added to Scheduler CRD here https://github.com/openshift/api/pull/366

**- Description for the changelog**
